tsconfig.aliases.json

```diff
@@ -131,6 +131,7 @@
             "@desktop/*": ["./packages/suite-desktop/src/*"],
             "@desktop-electron/*": ["./packages/suite-desktop/src-electron/*"],
             "@trezor/utils": ["./packages/utils/src"],
+            "@trezor/analytics": ["./packages/analytics/src"],
```

Comment:
this is not how packages should be used, just add them to `package.json` where you want to use them and call `yarn refs` 

check the docs here `docs/packages/creating-packages.md`

Reply:
`            "@trezor/utils": ["./packages/utils/src"],
` is correctly there or not?